freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/upgrade/ccm/UpgradeCcmState.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.freeipa.flow.stack.upgrade.ccm;
+
+import com.sequenceiq.flow.core.FlowState;
+import com.sequenceiq.flow.core.RestartAction;
+import com.sequenceiq.freeipa.flow.FillInMemoryStateStoreRestartAction;
+
+public class UpgradeCcmState implements FlowState {
```

Comment:
could you use an enum as we usually do?

Reply:
I can but I wouldn't. With this I was able to use literals to name the action beans. Enum had no addition value in the flow. but you can convince me.

Reply:
I see, but if this would be an enum, you could do it with the other way around, like:
```
public static final String UPGRADE_CCM_CHECK_PREREQUISITES_STATE_NAME = UPGRADE_CCM_CHECK_PREREQUISITES_STATE.name();
```
what do you think?

Reply:
unfortunately this is not work:
`public static final String UPGRADE_CCM_CHECK_PREREQUISITES_STATE_NAME = UPGRADE_CCM_CHECK_PREREQUISITES_STATE.name();`
since this is not resulting as constant, and I can't use it in annotation

Reply:
alright, tanks for checking. if you find this better I can accept. although it's a bit different of how we usually implement flows which might cause a bit of confusion later